turbo/snapshotsync/snapshot_builder.go

```diff
@@ -22,12 +21,251 @@ import (
 	"github.com/ledgerwatch/erigon/log"
 )
 
-//What number should we use?
+//maxReorgDepth max reorg depth. We should create snapshot after it
 const maxReorgDepth = 90000
```

Comment:
There is constant called `params.FullImmutabilityThreshold` which is equal to that number

Reply:
fixed